### DISCUSSION TITLE
SegmentedButton multiSelect

### DISCUSSION THREAD

**Discussion by nmateo** [callstack/react-native-paper#3624] [Q&A] [answered]
Hello, i want to use the segmentedButtons props "multiSelect" so i could select multiple buttons a the same time, but can't seem to find documentation and i did not manage to get it to work, thanks

**Comment by lukewalczak** [comment, marked as answer]
Please check the following snack: https://snack.expo.dev/@react-native-paper/segmentedbuttons-multiselect
